src/Common/src/Common.Expression/Spring/Ast/FunctionReference.cs

```diff
@@ -0,0 +1,163 @@
+            {
+                return false;
+            }
+
+            if (!_method.IsStatic || !_method.IsPublic || !_method.DeclaringType.IsPublic)
+            {
+                return false;
+            }
+
+            foreach (var child in _children)
+            {
+                if (!child.IsCompilable())
+                {
+                    return false;
+                }
+            }
+
+            return true;
+        }
+
+        public override void GenerateCode(DynamicMethod mv, CodeFlow cf)
+        {
+            // Method method = this.method;
+            // Assert.state(method != null, "No method handle");
+            // String classDesc = method.getDeclaringClass().getName().replace('.', '/');
+            // generateCodeForArguments(mv, cf, method, this.children);
+            // mv.visitMethodInsn(INVOKESTATIC, classDesc, method.getName(),
+            //        CodeFlow.createSignatureDescriptor(method), false);
+            // cf.pushDescriptor(this.exitTypeDescriptor);
+        }
+
+        private object[] GetArguments(ExpressionState state)
+        {
+            // Compute arguments to the function
+            var arguments = new object[ChildCount];
+            for (var i = 0; i < arguments.Length; i++)
+            {
+                arguments[i] = _children[i].GetValueInternal(state).Value;
+            }
+
+            return arguments;
+        }
+
+        private TypedValue ExecuteFunctionJLRMethod(ExpressionState state, MethodInfo method)
+        {
+            var functionArgs = GetArguments(state);
+
+            if (!method.IsVarArgs())
+            {
+                var declaredParamCount = method.GetParameters().Length;
+                if (declaredParamCount != functionArgs.Length)
+                {
+                    throw new SpelEvaluationException(SpelMessage.INCORRECT_NUMBER_OF_ARGUMENTS_TO_FUNCTION, functionArgs.Length, declaredParamCount);
+                }
+            }
+
+            if (!method.IsStatic)
+            {
+                throw new SpelEvaluationException(StartPosition, SpelMessage.FUNCTION_MUST_BE_STATIC, ClassUtils.GetQualifiedMethodName(method), _name);
+            }
+
+            // Convert arguments if necessary and remap them for varargs if required
+            var converter = state.EvaluationContext.TypeConverter;
+            var argumentConversionOccurred = ReflectionHelper.ConvertAllArguments(converter, functionArgs, method);
+            if (method.IsVarArgs())
+            {
+                functionArgs = ReflectionHelper.SetupArgumentsForVarargsInvocation(ClassUtils.GetParameterTypes(method), functionArgs);
+            }
+
+            var compilable = false;
+            try
+            {
+                var result = method.Invoke(method.GetType(), functionArgs);
+                compilable = !argumentConversionOccurred;
+                return new TypedValue(result, result == null ? method.ReturnType : result.GetType());
```

Comment:
```suggestion
                return new TypedValue(result, result?.GetType() ?? method.ReturnType);
```

Reply:
This is for cosmetics/readability, it is not the only case and is completely optional